deploy/index.js

```diff
@@ -0,0 +1,374 @@
+    const latestIdx = filtered.findIndex(
+      (release) => release.name === latestTagName
+    );
+    if (latestIdx !== -1) {
+      writeFileSync(
+        LATEST_FILE_NAME,
+        JSON.stringify(transformedToS3[latestIdx], null, 2)
+      );
+      identifyLatestSuccess = true;
+    } else {
+      console.err(
+        `Could not find latest release version tag in releases ${latestTagName}.`
+      );
+    }
+  }
+  //  write the first version in the array !IMPORTANT - releases are sorted in order of release
+  if (!identifyLatestSuccess) {
+    if (filtered.len > 0) {
+      writeFileSync(
+        LATEST_FILE_NAME,
+        JSON.stringify(transformedToS3[0], null, 2)
+      );
+      identifyLatestSuccess = true;
+    } else {
+      console.error(
+        "All features were filtered, or not found in response. Unable to find latest release version."
+      );
+    }
+  }
+  let ret = {
+    fileWriteSuccess: identifyLatestSuccess,
+    releaseData: filtered,
+  };
+  return ret;
+};
+
+/**
+ * Upload manifest and JSON file
+ *
+ */
+const uploadManifestAndLatest = () => {
+  tryPut(MANIFEST_FILE_NAME, MIME_JSON);
+  tryPut(LATEST_FILE_NAME, MIME_JSON);
+};
+
+/**
+ * Upload Assets
+ *
+ */
+const uploadReleaseAssets = (assets) => {
+  for (let asset of assets) {
+    const folderName = asset.name;
+    // upload Jar
+    tryPutRemote(
+      asset.jar.download_url,
+      asset.jar.name,
+      folderName,
+      "application/java-archive"
+    );
+
+    // upload configuration
+    tryPutRemote(
+      asset.configuration.download_url,
+      asset.configuration.name,
+      folderName,
+      MIME_JSON
+    );
+
+    // upload configuration
+    tryPutRemote(
+      asset.jvmRun.download_url,
+      asset.jvmRun.name,
+      folderName,
+      "text/plain"
+    );
+  }
+};
+
+/**
+ * Send email with summary via SendGrid
+ *
+ */
+const sendNotification = () => {
+  const file_content = readFileSync("./index.js", "utf-8");
+  const content = Buffer.from(file_content).toString("base64");
+  const target_emails = process.env.SENDGRID_TARGET_LIST;
+  let to_list;
+
+  sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+  if (target_emails === undefined) {
+    console.error(
+      "Notification canceled because SENDGRID_TARTGET_LIST environment var was not set."
+    );
+    return;
+  }
+  try {
+    to_list = JSON.parse(process.env.SENDGRID_TARGET_LIST);
+  } catch (error) {
+    console.error(`JSON parsing error on string ${target_emails}`);
+    console.error(error);
+    return;
+  }
+
+  const msg = {
+    to: to_list,
+    from: "no-reply@hmsamericas.com", // Change to your verified sender
+    subject: "test!",
+    html: "<div><p>Greetings !!</p><p>This is an automated email. The Flexy Cumulocity Connector S3 repo has been updated. The manifest file is attached to this email.</p></div>",
+    attachments: [
+      {
+        content: content,
+        filename: "manifest.json",
+        type: "application/json",
```

Comment:
```suggestion
        type: "MIME_JSON",
```
Use the MIME constants

Reply:
changed